### ISSUE TITLE
W0613 unused-argument false positive

### ISSUE THREAD

**Issue by teake** [PyCQA/pylint#3444]
### Steps to reproduce
1.  Make a Python file (`w0613.py`) with the following content:
```
def square(x):
    return x * x

```

2. Run `pylint --disable=all --enable=W0613 w0613.py`

### Current behavior

Output:
```
************* Module w0613
w0613.py:1:11: W0613: Unused argument 'x' (unused-argument)

------------------------------------------------------------------
Your code has been rated at 5.00/10 (previous run: 5.00/10, +0.00)
```

### Expected behavior

That W0613 isn't raised.

### pylint --version output

This is on the master version of both pylint and astroid at the time of writing.

```
pylint 2.5.0-dev1
astroid 2.4.0
Python 3.7.6 | packaged by conda-forge | (default, Mar  5 2020, 15:03:29) 
[Clang 9.0.1 ]
```

**Comment by teake** [comment]
Just tested it, and looks like c5181886b59799740306829e0e21e548c50db391 also closes this one. Thanks again!
